rust/kcl-lib/src/parsing/ast/types/mod.rs

```diff
@@ -399,6 +403,40 @@ impl Node<Program> {
         Ok(new_program)
     }
 
+    pub fn change_experimental_features(&self, warning_level: Option<WarningLevel>) -> Result<Self, KclError> {
```

Comment:
I would be great to create a util fn to abstract these two types of lookups and insertion and share with change_default_units, but lacking energy to get it done as part of this pass haha